web/src/regression/core.test.ts

```diff
@@ -124,10 +124,13 @@ describe('Core functionality regression test suite', () => {
             )
         }
 
+        // When you type (or paste) "{" into the empty user settings editor it adds a "}". That's why
+        // we cannot type all the previous text, because then we would have two "}" at the end.
+        const textToTypeFromPrevious = previousSettings.substring(0, previousSettings.indexOf('\n}'))
```

Comment:
If your intention is to strip the `}` only at the end, this would be more reliable to not accidentally strip a `}` in the middle of the text.
(also we should really address #5259)
```suggestion
        const textToTypeFromPrevious = previousSettings.replace(/\}$/, '')
```